src/utils.py

```diff
@@ -0,0 +1,49 @@
+"""Utility classes and functions."""
+import re
+
+import daiquiri
+import requests
+
+from src.config import DAIQUIRI_LOG_LEVEL
+from src.gremlin import execute_query
+
+daiquiri.setup(level=DAIQUIRI_LOG_LEVEL)
+log = daiquiri.getLogger(__name__)
+
+
+def sanitize_text_for_query(text):
+    """Sanitize text so it can used in queries."""
+    if text is None:
+        return ''
+
+    if not isinstance(text, str):
+        raise ValueError(
+            'Invalid query text: expected string, got {t}'.format(t=type(text))
+        )
+
+    # TODO - As of now bypassing till we will have model defined and working fine in environment.
+    strict_check_words = ['drop', 'delete', 'update', 'remove', 'insert']
+    if re.compile('|'.join(strict_check_words), re.IGNORECASE).search(text):
```

Comment:
we can just do `if text.lower() in strict_check_words.lower:` here, no need to use a RE.

Reply:
done